dockers/docker-fpm-frr/bgpcfgd

```diff
@@ -290,6 +292,34 @@ class BGPPeerMgr(Manager):
         vrf, nbr = key.split('|', 1)
         if key not in self.peers:
             cmd = None
+
+            if "local_addr" not in data:
+                syslog.syslog(syslog.LOG_WARNING, 'Peer {}. Error in missing required attribute "local_addr"'.format(key))
+            else:
+                # The route that belongs to a vnet cannot be advertised by the default BGP session.
```

Comment:
I do not know why you talk about *route* here. data['local_addr'] is the bgp session's local IP. Here you just check if bgp session's local IP is in one of local interface subnet.